_config.yml

```diff
@@ -10,6 +10,10 @@ title: This is the site title
 # See https://github.com/bryanbraun/anchorjs for more information.
 # anchor_js_targets: [h1, h2, h3, h4, h5, h6]
 
+# Configuration stylesheets
```

Comment:
Thinking we should open a new issue to have a default main.css set up and ready to go as most folks will want to make some sort of modifications to the styles.

Reply:
I think we should consider the best way to do this, because these config options do not fall through to the actual jekyll instance using the theme automatically, so this would require folks to manually copy/paste this `styles` value into their own `_config.yml`. To avoid that I think the best option might be to modify `includes/styles.html` instead. I did something similar on the theme_variant branch I am working on, so maybe I can get that merged in first and that'll take care of this?

Reply:
That sounds like good approach 👍